contracts/examples/multisig/interaction/Multisig-adder-egld.erdjs.md

```diff
@@ -0,0 +1,72 @@
+# Multisig
+
+First [set up a node terminal](../../../../tutorial/src/interaction/interaction-basic.md).
+
+```javascript
+let erdjs = await require('@elrondnetwork/erdjs');
+let { erdSys, Egld, wallets: { alice, bob, carol }} = await erdjs.setupInteractive("local-testnet");
+
+let multisig = await erdSys.loadWrapper("contracts/examples/multisig");
+
+// Deploy a multisig contract with a quorum of 3, but 4 possible signers: alice, bob, carol, dan
+// A quorum is the minimum required number of signers in order to be able to perform a proposed action
```

Comment:
new notion => write it in the docs

Reply:
This info is already in the readme for this contract. I left a note at the top of this script pointing to that readme.